ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -302,24 +319,30 @@ runDataDiffusion tracers
                Left sd -> do
                    a <- Socket.getSocketName sd
                    case a of
-                        (Socket.SockAddrUnix path) ->
+                        (Socket.SockAddrUnix path) -> do
+                          traceWith dtDiffusionInitializationTracer $ DiffusionInitiatilizationCreatingLocalServerSocketForUnixPath path
                           return (sd, Snocket.localSnocket iocp path)
-                        _                          ->
+                        unsupportSocket -> do
                             -- TODO: This should be logged.
+                            traceWith dtDiffusionInitializationTracer $ DiffusionInitiatilizationUnsupportedSocketType (show unsupportSocket)
```

Comment:
I would call this `UnsupportedSystemdLocalSocket (unsupportedAddr  :: SockAddr)`